doc/source/getting-started-linux-tutorial.rst

```diff
@@ -14,33 +14,35 @@ Volatility3 does not provide the ability to acquire memory.  Below are some exam
 
 Procedure to create symbol tables for linux
---------------------------------------------
+-------------------------------------------
 
-To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.
-
-.. tip:: It may be possible to locate pre-made ISF files from the `Linux ISF Server <https://isf-server.techanarchy.net/>`_ ,
-    which is built and maintained by `kevthehermit <https://twitter.com/kevthehermit>`_.
-    After creating the file or downloading it from the ISF server, place the file under the directory ``volatility3/symbols/linux``.
-    If necessary create a linux directory under the symbols directory (this will become unnecessary in future versions).
+To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.  
+After creating the file, place it under the directory ``volatility3/symbols``.  
+Volatility3 will automatically detect and use symbol tables from this location.
 
 
 Listing plugins
 ---------------
 
-The following is a sample of the linux plugins available for volatility3, it is not complete and more plugins may
-be added.  For a complete reference, please see the volatility 3 :doc:`list of plugins <volatility3.plugins>`.
-For plugin requests, please create an issue with a description of the requested plugin.
+Volatility3 currently supports over 40 Linux-specific plugins covering a wide range of forensic analysis needs, such as process enumeration, memory-mapped file inspection, loaded modules, and kernel tracing features.
+
+Some representative plugins include:
+
+- ``linux.pslist``: Lists running processes with their PIDs and PPIDs.
+- ``linux.bash``: Recovers bash command history from memory.
+- ``linux.lsmod``: Displays loaded kernel modules.
+- ``linux.kmsg``: Reads messages from the kernel log buffer.
+- ``linux.elfs``: Lists all memory-mapped ELF files.
+- ``linux.check_creds``: Checks for suspicious credential structures.
+- ``linux.vmayarascan``: Scans process memory using YARA signatures.
+
+For a full list of supported plugins, run the following command:
 
 .. code-block:: shell-session
 
-    $ python3 vol.py --help | grep -i linux. | head -n 5
-        banners.Banners     Attempts to identify potential linux banners in an
-        linux.bash.Bash     Recovers bash command history from memory.
-        linux.malware.check_afinfo.Check_afinfo
-        linux.malware.check_creds.Check_creds
-        linux.malware.check_idt.Check_idt
+    $ python3 vol.py --help | grep -i linux.
 
-.. note:: Here the command is piped to grep and head to provide the start of the list of linux plugins.
+.. note:: You can also filter and inspect available plugins using more sophisticated patterns or tools like ``grep``, ``awk``, or simply explore the source under ``volatility3/framework/plugins/linux``.
```

Comment:
Actually, This path (`volatility3/plugins/linux`) is also included result.

Reply:
Thank you for the insights! I agree with @ikelos that volatility3/framework/plugins/linux is the official plugin path, and I believe it's sufficient to reference only this location in the documentation to avoid confusion. Since the other path is not officially used or shipped, I think no changes are necessary at this time.